### PR TITLE
optimize: add Installation section in README.markdown, add warning message under the install target

### LINKED ISSUE #449: why the lua version number is commented out in Makefile
Can someone explain why the lua version number is commented out in Makefile?
I think it would be better if we keep it

> OPENRESTY_PREFIX=/usr/local/openresty
> 
> #LUA_VERSION := 5.1
> PREFIX ?=          /usr/local
> LUA_INCLUDE_DIR ?= $(PREFIX)/include
> LUA_LIB_DIR ?=     $(PREFIX)/lib/lua/$(LUA_VERSION)
> 	@@ -12,9 +13,11 @@ all: ;
>

### PR DESCRIPTION
**Pull request by lynch1981**
fix #449
I hereby granted the copyright of the changes in this pull request
to the authors of this lua-resty-core project.

### CODE REVIEW

**Review by zhuizhuhaomeng — APPROVED** [review]

**Comment by zhuizhuhaomeng** [comment]
This is not a good idea.
It breaks the backward compatibility.
We should change the installation command in the readme.

**Comment by lynch1981** [comment]
> This is not a good idea. It breaks the backward compatibility. We should change the installation command in the readme.
got it

**Comment by lynch1981** [comment]
The Installation part is missing in README.markdown, please review the section I added @zhuizhuhaomeng

File: Makefile [hunk 1]
```diff
@@ -1,9 +1,9 @@
 OPENRESTY_PREFIX=/usr/local/openresty
 
-LUA_VERSION := 5.1
+#LUA_VERSION := 5.1
 PREFIX ?=          /usr/local
 LUA_INCLUDE_DIR ?= $(PREFIX)/include
-LUA_LIB_DIR ?=     $(PREFIX)/share/lua/$(LUA_VERSION)
+LUA_LIB_DIR ?=     $(PREFIX)/lib/lua/$(LUA_VERSION)
```
Comment (zhuizhuhaomeng) [inline Makefile]: Let's keep the original LUA_LIB_DIR and comment out the LUA_VERSION.
Instead we add an echo warning message under the install target.

**Review by zhuizhuhaomeng — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (lynch1981) [reply Makefile]: good idea

**Comment by lynch1981** [comment]
Thanks to @zhuizhuhaomeng for the detailed description of the installation chapter

**Pull request closed without merging**